Firmware/odrive-interface.yaml

```diff
@@ -337,7 +380,6 @@ interfaces:
           BrakeDeadtimeViolation:
           UnexpectedTimerCallback:
           CurrentSenseSaturation:
-          InverterOverTemp:
           CurrentLimitViolation:
```

Comment:
again to reduce the likelihood of silent errors you can "pin" the bit number of the succeeding flag: `CurrentLimitViolation: {bit: 12}`

Reply:
Fixed.